java/bench/hive/src/java/org/apache/orc/bench/hive/rowfilter/BooleanRowFilterBenchmark.java

```diff
@@ -91,10 +91,14 @@ public void readOrcNoFilter(Blackhole blackhole, InputState state) throws Except
    * Run this test:
    *  java -cp hive/target/orc-benchmarks-hive-*-uber.jar org.apache.orc.bench.hive.rowfilter.BooleanRowFilterBenchmark
    */
-  public static void main(String[] args) throws RunnerException {
-    new Runner(new OptionsBuilder()
-        .include(BooleanRowFilterBenchmark.class.getSimpleName())
-        .forks(1)
-        .build()).run();
+  public static void main(String[] args) {
```

Comment:
Do you know why this change is required? Apache ORC code follows the official JMHSample file.
- https://github.com/openjdk/jmh/blob/cb3c3a90137dad781a2a37fda72dc11ebf253593/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_01_HelloWorld.java#L95

```java
    public static void main(String[] args) throws RunnerException {
        Options opt = new OptionsBuilder()
                .include(JMHSample_01_HelloWorld.class.getSimpleName())
                .forks(1)
                .build();

        new Runner(opt).run();
    }
```

Reply:
Because these Benchmarks inherit `extends org.openjdk.jmh.Main`, `org.openjdk.jmh.Main` removed the method throwing `RunnerException` in `1.23`.

https://github.com/openjdk/jmh/commit/4264de9486c32b48da8161e3ac076a0187b4176f#diff-30878bccaff9f11a64415de78f1feebe304334a314d8bb6f8c7bc50bd21aee19L39

Reply:
Does it mean the official `JMHSample_01_HellowWorld.java` is also broken?

Reply:
`JMHSample_01_HelloWorld` does not inherit jmh.Main ( `extends org.openjdk.jmh.Main` )

Reply:
Oh, got it. Thanks. Now, I understand.